code/modules/antagonists/changeling/changeling.dm

```diff
@@ -977,6 +985,7 @@
 	var/underwear_color
 	var/undershirt_color
 	var/socks_color
+	var/bra_color
```

Comment:
Modularize this, it doesn't need to be defined here (might as well move this whole bunch as well, while you're at it).